### DISCUSSION TITLE
Massive build times

### DISCUSSION THREAD

**Discussion by demberto** [microsoft/react-native-windows#10097] [General]
Just created a new project, and its already taken an hour to build the Windows distribution. I have 8GB RAM and an i5 6th gen desktop

**Comment by 527395632** [comment]
The first build is very time-consuming, because c++ code compilation consumes a lot of CPU. When I compile i7-11700, it will be 100% and last for more than 10 minutes. Then the project folder will become very large because a large number of files are output when c++ is compiled Obj file. It is recommended to use vscode to run the project after the first compilation, which will be much faster. Personal experience can be used for reference

**Reply by demberto** [reply]
Compared to something like flutter this is pure trash. Anyways MS is moving over to good ol' Win32 from UWP with Windows App SDK. I waited for a hour and half yesterday and it was still compiling, and it was possibly stuck. *What is with this tons of trash frameworks MS is coming up with on like a daily basis, anyways?*

**Comment by asklar** [comment]
@demberto You should be able to use the `useExperimentalNuget` flag in the CLI to create an app that uses the prebuilt Microsoft.ReactNative binaries, which will remove a large part of your build times.

**Reply by 527395632** [reply]
Thank you for your help, and then check the documentation. I did not find the 'useexperimentalnuget' option in my own test. I don't know how to implement it? Can you write down the detailed command line? thank you

**Reply by 527395632** [reply]
Sorry, I didn't read the document carefully. I have found the answer to this question. He is“ https://microsoft.github.io/react-native-windows/docs/NuGet ”Instead of on the command page of "react native windows cli". If you have similar questions, you can also refer to my reply here. I hope it can help you
